.github/workflows/publish.yml

```diff
@@ -0,0 +1,116 @@
+# https://docs.github.com/en/actions/creating-actions/creating-a-composite-action
+name: Publish binaries
+
+on:
+  pull_request:
+    types: [opened, synchronize, closed]
+    branches:
+      - "main"
+      - "master"
+  workflow_dispatch:
+    inputs:
+      distro:
+        description: Distro
+        required: true
+        type: choice
+        options:
+          - debian
+          - alpine
+      arch:
+        description: Arch
+        required: true
+        type: choice
+        options:
+          - aarch64
+          - amd64
+          - armv6hf
+          - armv7hf
+          - i386
+      node-version:
+        description: Node version
+        required: true
+        type: string
+      no-intl:
+        description: 'Build with --with-intl=none'
+        required: false
+        type: boolean
+        default: false
+
+env:
+  DISTRO: ${{ inputs.distro || 'debian' }}
+  ARCH: ${{ inputs.arch || 'aarch64' }}
+  NODE_VERSION: ${{ inputs.node-version || 'v19.6.1' }}
```

Comment:
Should we actually always require those an fail if not provided?
This way we no longer have hardcode the node versions that we have marked as latest in the base image contacts (atm v19.6.1).

Reply:
The only reason I provided fallback values here is so I could test the workflow on pull requests, when there are no inputs provided at all. I agree that when triggered manually those fields should be required.

Also, this was the minimum port from Jenkins to unblock us. A proper workflow would have a build matrix for all supported versions and run automatically on merge IMO.